Drop-In/src/test/java/com/braintreepayments/api/DropInClientUnitTest.java

```diff
@@ -1840,4 +1855,13 @@ private Configuration mockConfiguration(boolean paypalEnabled, boolean venmoEnab
 
         return configuration;
     }
+
+    private static BrowserSwitchResult createSuccessfulBrowserSwitchResult() {
+        int requestCode = 123;
+        Uri url = Uri.parse("www.example.com");
+        String returnUrlScheme = "sample-scheme";
+        BrowserSwitchRequest browserSwitchRequest = new BrowserSwitchRequest(
+                requestCode, url, new JSONObject(), returnUrlScheme, true);
```

Comment:
```suggestion
                requestCode, url, new JSONObject(), returnUrlScheme);
```

Same thing here - I think there may be a few things off in these tests after rebasing. They don't compile locally for me